CHANGES.md

```diff
@@ -1,5 +1,71 @@
+- [dagit] The asset "Materialize Partitions" modal now presents an error when partition config or tags cannot be generated.
+- [dagit] The right sidebar of the global asset graph no longer defaults to 0% wide in fresh / incognito browser windows, which made it difficult to click nodes in the global graph.
+- [dagit] In the asset catalog, the search bar now matches substrings so it's easier to find assets with long path prefixes.
+- [dagit] Dagit no longer displays duplicate downstream dependencies on the Asset Details page in some scenarios.
+- [dagster-fivetran] Assets created using `build_fivetran_assets` will now be properly tagged with a `fivetran` pill in Dagit.
+
+### Bugfixes
+
+- Fixed issue causing step launchers to fail in many scenarios involving re-execution or dynamic execution.
+- Previously, incorrect selections (generally, step selections) could be generated for strings of the form `++item`. This has been fixed.
+- Fixed an issue where run status sensors sometimes logged the wrong status to the event log if the run moved into a different status while the sensor was running.
+- Fixed an issue where daily schedules sometimes produced an incorrect partition name on spring Daylight Savings time boundaries.
+- [dagit] Certain workspace or repo-scoped pages relied on versions of the `SQLAlchemy` package to be `1.4` or greater to be installed. We are now using queries supported by `SQLAlchemy>=1.3`. Previously we would raise an error including the message: `'Select' object has no attribute 'filter'`.
+- [dagit] Certain workspace or repo-scoped pages relied on versions of `sqlite` to be `3.25.0` or greater to be installed. This has been relaxed to support older versions of sqlite. This was previously marked as fixed in our `0.14.0` notes, but a handful of cases that were still broken have now been fixed. Previously we would raise an error (`sqlite3.OperationalError`).
+- [dagit] The launchpad is significantly more responsive when you're working with very large partition sets.
+- [dagit] We now show an informative message on the Asset catalog table when there are no matching assets to display. Previously, we would show a blank white space.
+- [dagit] When changing presets / partitions in the launchpad, Dagit preserves user-entered tags and replaces only the tags inherited from the previous base.
+- [dagit] Running Dagit without a backfill daemon no longer generates a warning unless queued backfills are present. Similarly, a missing sensor or schedule daemon only yields a warning if sensors or schedules are turned on.
+- [dagit] On the instance summary page, hovering over a recent run’s status dot shows a more helpful tooltip.
+- [dagit] Dagit no longer hangs when rendering the run gantt chart for certain graph structures.
+- [dagster-airbyte] Fixed issues that could cause failures when generating asset materializations from an Airbyte API response.
```

Comment:
* [dagster-aws] 0.14.3 removed the ability for the `EcsRunLauncher` to use sidecars without you providing your own custom task definition. Now, you can continue to inherit sidecars from the launching task’s task definition by setting `include_sidecars: True` in your run launcher config.